### ISSUE TITLE
[ICON REQUEST]: Affinity Designer/Photo/Publisher (all 3 Affinity products)

### ISSUE THREAD

**Issue by grandslammer** [devicons/devicon#1259]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Excellent Adobe Illustrator, Photoshop and InDesign alternatives that is quite popular

### Links and sources

https://affinity.serif.com/en-gb/

I strongly recommend these products as they are on-par with their Adobe equivalents for a fraction of the price. One-time payment with free updates for life.

It would be great to have these logos included!

**Comment by BenSouchet** [comment]
Hi, according to the guideline :
https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F

The affinity softwares does not enter in one of the category for devicon, since this is not related to code / dev.

**Comment by grandslammer** [comment]
> Hi, according to the guideline : [devicons/devicon/wiki/What-Icons-Do-We-Accept%3F](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F)
> 
> The affinity softwares does not enter in one of the category for devicon, since this is not related to code / dev.

I think that they do fall under the special cases > related fields category - see [Special Cases](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#special-cases-see-this-discussion-for-more-details)

**Comment by BenSouchet** [comment]
It will take more explanations / arguments to convince me on that.

**Comment by Snailedlt** [comment]
I agree with @BenSouchet here. But feel fre to try and convince us otherwise

**Comment by grandslammer** [comment]
Adobe products fall under the special cases category.
With that in mind, we all understand that developers are generally not also expected to do graphic design.

However, there are times when a developer may have to do some basic editing. I can think of an example I had only this week with a website I am currently developing, Some of the assets I was given needed to have some minor changes/edits to them. Luckily, I had Affinity Designer installed and was able to make the changes myself quickly and easily.

If I did not have the tools and skills to do so, I would have had to send a whole thread of emails back and forth to the original designer (who is no longer part of the project for over a year), expecting them to make the changes, checking and verifying them on my end, replying to them again to say that they needed to change something else again, etc etc. It would have turned into a mess quite quickly.

The beauty about Affinity from a developer's POV is that it is a lifetime purchase, which means that developers get an excellent deal if they ever have to use the product even on the rare occasion. No monthly subscriptions for something you barely ever use - but when you do need to use it, it's very handy to have a tool there ready to go.

€64.99 for Affinity Designer is a great deal in the grand scheme of things, and is a very handy tool for any front end developer to have in their toolkit. It easily rivals Illustrator in terms of functionality, and can be a life-saver at times.

**Comment by lunatic-fox** [comment]
@grandslammer is right about the category. However, I agree with @BenSouchet and @Snailedlt, since those tools are more related to design than to development.

I think they are great tools, but both artists and developers can only use them as designer tools in their respective use cases. That way, even if you are doing an UI for some app, for instance, you still need another tool to add the functionality (development).

Just for clarification:
`aftereffects`, `illustrator` and `inkscape`, are similar icon exceptions that were made before [**What Icons Do We Accept?**](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F) wiki.

Open to discussion! 😃

**Comment by Snailedlt** [comment]
Just for clarity. This falls under the following [special case](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#special-cases-see-this-discussion-for-more-details):

> - Related fields like graphic designs or game development. Since many "development" jobs require people to know related fields, some software will be accepted in Devicon even though they aren't strictly "development tools".
>    - ex. some Adobe products (Photoshop, Aftereffects), game engines (Unity, Unreal), CMS (Shopify), etc... See [this](https://github.com/devicons/devicon/pull/336) for more details.

With that in mind, this is pretty much up to what the community as a whole decides... so let the discussion commence! 🗣️

**Comment by Panquesito7** [comment]
I think this could actually fit in the repository. If you check the repository description, it says `designing`, which these tools' categories fall on.
IMO, we can add these to the repository. Let's just wait and see what other people think. 🙂

Should we create a discussion for this or discuss in the issue?

**Comment by Snailedlt** [comment]
Let's keep it simple, and keep the discussion here :)
